File: bot (public)/bot.jl
import Pkg
Pkg.add("PyCall")
Pkg.add("JSON")
using PyCall
using JSON
using Dates

Southx = pyimport("Southxchange")
Southx.Southxchange("Private", "Secret") #Add your API keys here
Market = Southx.Market()
Markets = Southx.Markets()
Wallets = Southx.Wallets()

function getprice()
    Markets.price("yec", "btc")["Ask"]
end

function getbalances()
    JSON.parse(Wallets.balances())
end

function buy()
    price = getprice()
    Market.placeorder("yec", "btc", amount=1, type="BUY", limitprice=price)
    sleep(1)
    println(getbalances())
end

while 1 == 1
    buy()
    sleep(3600)
end

File: bot (public)/bot.py
#!/bin/python3
import Southxchange
import json
import time

Southxchange.Southxchange('Private', 'Secret') #Use your API keys here
Market = Southxchange.Market()
Markets = Southxchange.Markets()
Wallets = Southxchange.Wallets()
time.sleep(1)

print(Markets.price('yec', 'btc'))

def getbalances():
    print(json.loads(Wallets.balances()))


getbalances()
time.sleep(1)

def buy():
    price = Markets.price('yec', 'btc')['Ask']
    Market.placeorder('YEC', 'BTC', amount=1, type='BUY', limitprice=price)
    time.sleep(1)
    getbalances()

a = 1
while a == 1:
    buy()
    time.sleep(3600)